vector/v.info/local_proto.h

```diff
@@ -17,7 +17,7 @@ void parse_args(int, char **, char **, char **, int *, int *, int *,
                 enum OutputFormat *);
 
 /* print.c */
-void format_double(double, char *);
+void format_double(double, char[256]);
```

Comment:
The print.c file of v.info, defined a new BUFSZ macro with the same 256 value. Is it because having the macro in the header would cause clashes if included elsewhere?

Reply:
Good catch. Better to use the macro, no reason not to use it here. Updated.